Installing on MacOS M3

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

None

### Describe the Bug

Running brew install sunshine exits with:

../../../../build/assets/web/assets/ResourceCard-ea4a7cba.js                  1.67 kB │ gzip:   0.64 kB
../../../../build/assets/web/assets/index-1d511c0f.js                         2.06 kB │ gzip:   0.90 kB
../../../../build/assets/web/assets/Navbar-dbaf0800.js                        2.16 kB │ gzip:   0.94 kB
../../../../build/assets/web/assets/config-f4fb6dcb.js                        4.97 kB │ gzip:   2.10 kB
../../../../build/assets/web/assets/apps-966a1e70.js                         85.57 kB │ gzip:  25.93 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-fe085d29.js   327.84 kB │ gzip: 111.98 kB
✓ built in 1.04s
[ 98%] Built target web-ui
make: *** [all] Error 2

### Expected Behavior

Success

### Additional Context

command sunshine doesn't register since the build fails
Link given when failed /lizardbyte/homebrew-homebrew/issues does not exist (issue reporting disabled for that repo)

### Host Operating System

macOS

### Operating System Version

Sonoma 14.6.1

### Architecture

arm

### Sunshine commit or version

the one on homebrew, can't run the command

### Package

Linux - Homebrew

### GPU Type

AMD

### GPU Model

Apple Silicon M3 Pro Max

### GPU Driver/Mesa Version

Apple Silicon M3 Pro Max

### Capture Method

None

### Config

```Shell
none
```


### Apps

_No response_

### Relevant log output

```shell
../../../../build/assets/web/assets/ResourceCard-ea4a7cba.js                  1.67 kB │ gzip:   0.64 kB
../../../../build/assets/web/assets/index-1d511c0f.js                         2.06 kB │ gzip:   0.90 kB
../../../../build/assets/web/assets/Navbar-dbaf0800.js                        2.16 kB │ gzip:   0.94 kB
../../../../build/assets/web/assets/config-f4fb6dcb.js                        4.97 kB │ gzip:   2.10 kB
../../../../build/assets/web/assets/apps-966a1e70.js                         85.57 kB │ gzip:  25.93 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-fe085d29.js   327.84 kB │ gzip: 111.98 kB
✓ built in 1.04s
[ 98%] Built target web-ui
make: *** [all] Error 2
```

Thank you!
This gives the following error

==> Installing sunshine-beta from lizardbyte/homebrew
==> Building docs: enabled
==> Statically linking Boost libraries
==> Linking against ICU libraries at: /opt/homebrew/opt/icu4c/lib
==> cmake -S . -B build -DBUILD_WERROR=ON -DHOMEBREW_ALLOW_FETCHCONTENT=ON -DOPE
==> make
Last 15 lines from /[redacted]

Patching output file 1043/1052
Patching output file 1044/1052
Patching output file 1045/1052
Patching output file 1046/1052
Patching output file 1047/1052
Patching output file 1048/1052
Patching output file 1049/1052
Patching output file 1050/1052
Patching output file 1051/1052
Patching output file 1052/1052
type lookup cache used 6335/65536 hits=14996 misses=7844
symbol lookup cache used 4945/65536 hits=27893 misses=4945
finished...
[ 75%] Built target docs
make: *** [all] Error 2

```
Running dot for graph 1/747
Running dot for graph 2/747
Running dot for graph 3/747
Running dot for graph 4/747
Running dot for graph 5/747
Running dot for graph 6/747
Running dot for graph 7/747
Running dot for graph 8/747
Running dot for graph 9/747
Running dot for graph 10/747
Running dot for graph 11/747
Running dot for graph 12/747
Running dot for graph 13/747
Running dot for graph 14/747
Running dot for graph 15/747
Running dot for graph 16/747
Running dot for graph 17/747
Running dot for graph 18/747
Running dot for graph 19/747
Running dot for graph 20/747
Running dot for graph 21/747
Running dot for graph 22/747
Running dot for graph 23/747
Running dot for graph 24/747
Running dot for graph 25/747
Running dot for graph 26/747
Running dot for graph 27/747
Running dot for graph 28/747
Running dot for graph 29/747
Running dot for graph 30/747
Running dot for graph 31/747
Running dot for graph 32/747
Running dot for graph 33/747
Running dot for graph 34/747
Running dot for graph 35/747
Running dot for graph 36/747
Running dot for graph 37/747
Running dot for graph 38/747
Running dot for graph 39/747
Running dot for graph 40/747
Running dot for graph 41/747
Running dot for graph 42/747
Running dot for graph 43/747
Running dot for graph 44/747
Running dot for graph 45/747
Running dot for graph 46/747
Running dot for graph 47/747
Running dot for graph 48/747
Running dot for graph 49/747
Running dot for graph 50/747
Running dot for graph 51/747
Running dot for graph 52/747
Running dot for graph 53/747
Running dot for graph 54/747
Running dot for graph 55/747
Running dot for graph 56/747
Running dot for graph 57/747
Running dot for graph 58/747
Running dot for graph 59/747
Running dot for graph 60/747
Running dot for graph 61/747
Running dot for graph 62/747
Running dot for graph 63/747
Running dot for graph 64/747
Running dot for graph 65/747
Running dot for graph 66/747
Running dot for graph 67/747
Running dot for graph 68/747
Running dot for graph 69/747
Running dot for graph 70/747
Running dot for graph 71/747
Running dot for graph 72/747
Running dot for graph 73/747
Running dot for graph 74/747
Running dot for graph 75/747
Running dot for graph 76/747
Running dot for graph 77/747
Running dot for graph 78/747
Running dot for graph 79/747
Running dot for graph 80/747
Running dot for graph 81/747
Running dot for graph 82/747
Running dot for graph 83/747
Running dot for graph 84/747
Running dot for graph 85/747
Running dot for graph 86/747
Running dot for graph 87/747
Running dot for graph 88/747
Running dot for graph 89/747
Running dot for graph 90/747
Running dot for graph 91/747
Running dot for graph 92/747
Running dot for graph 93/747
Running dot for graph 94/747
Running dot for graph 95/747
Running dot for graph 96/747
Running dot for graph 97/747
Running dot for graph 98/747
Running dot for graph 99/747
Running dot for graph 100/747
Running dot for graph 101/747
Running dot for graph 102/747
Running dot for graph 103/747
Running dot for graph 104/747
Running dot for graph 105/747
Running dot for graph 106/747
Running dot for graph 107/747
Running dot for graph 108/747
Running dot for graph 109/747
Running dot for graph 110/747
Running dot for graph 111/747
Running dot for graph 112/747
Running dot for graph 113/747
Running dot for graph 114/747
Running dot for graph 115/747
Running dot for graph 116/747
Running dot for graph 117/747
Running dot for graph 118/747
Running dot for graph 119/747
Running dot for graph 120/747
Running dot for graph 121/747
Running dot for graph 122/747
Running dot for graph 123/747
Running dot for graph 124/747
Running dot for graph 125/747
Running dot for graph 126/747
Running dot for graph 127/747
Running dot for graph 128/747
Running dot for graph 129/747
Running dot for graph 130/747
Running dot for graph 131/747
Running dot for graph 132/747
Running dot for graph 133/747
Running dot for graph 134/747
Running dot for graph 135/747
Running dot for graph 136/747
Running dot for graph 137/747
Running dot for graph 138/747
Running dot for graph 139/747
Running dot for graph 140/747
Running dot for graph 141/747
Running dot for graph 142/747
Running dot for graph 143/747
Running dot for graph 144/747
Running dot for graph 145/747
Running dot for graph 146/747
Running dot for graph 147/747
Running dot for graph 148/747
Running dot for graph 149/747
Running dot for graph 150/747
Running dot for graph 151/747
Running dot for graph 152/747
Running dot for graph 153/747
Running dot for graph 154/747
Running dot for graph 155/747
Running dot for graph 156/747
Running dot for graph 157/747
Running dot for graph 158/747
Running dot for graph 159/747
Running dot for graph 160/747
Running dot for graph 161/747
Running dot for graph 162/747
Running dot for graph 163/747
Running dot for graph 164/747
Running dot for graph 165/747
Running dot for graph 166/747
Running dot for graph 167/747
Running dot for graph 168/747
Running dot for graph 169/747
Running dot for graph 170/747
Running dot for graph 171/747
Running dot for graph 172/747
Running dot for graph 173/747
Running dot for graph 174/747
Running dot for graph 175/747
Running dot for graph 176/747
Running dot for graph 177/747
Running dot for graph 178/747
Running dot for graph 179/747
Running dot for graph 180/747
Running dot for graph 181/747
Running dot for graph 182/747
Running dot for graph 183/747
Running dot for graph 184/747
Running dot for graph 185/747
Running dot for graph 186/747
Running dot for graph 187/747
Running dot for graph 188/747
Running dot for graph 189/747
Running dot for graph 190/747
Running dot for graph 191/747
Running dot for graph 192/747
Running dot for graph 193/747
Running dot for graph 194/747
Running dot for graph 195/747
Running dot for graph 196/747
Running dot for graph 197/747
Running dot for graph 198/747
Running dot for graph 199/747
Running dot for graph 200/747
Running dot for graph 201/747
Running dot for graph 202/747
Running dot for graph 203/747
Running dot for graph 204/747
Running dot for graph 205/747
Running dot for graph 206/747
Running dot for graph 207/747
Running dot for graph 208/747
Running dot for graph 209/747
Running dot for graph 210/747
Running dot for graph 211/747
Running dot for graph 212/747
Running dot for graph 213/747
Running dot for graph 214/747
Running dot for graph 215/747
Running dot for graph 216/747
Running dot for graph 217/747
Running dot for graph 218/747
Running dot for graph 219/747
Running dot for graph 220/747
Running dot for graph 221/747
Running dot for graph 222/747
Running dot for graph 223/747
Running dot for graph 224/747
Running dot for graph 225/747
Running dot for graph 226/747
Running dot for graph 227/747
Running dot for graph 228/747
Running dot for graph 229/747
Running dot for graph 230/747
Running dot for graph 231/747
Running dot for graph 232/747
Running dot for graph 233/747
Running dot for graph 234/747
Running dot for graph 235/747
Running dot for graph 236/747
Running dot for graph 237/747
Running dot for graph 238/747
Running dot for graph 239/747
Running dot for graph 240/747
Running dot for graph 241/747
Running dot for graph 242/747
Running dot for graph 243/747
Running dot for graph 244/747
Running dot for graph 245/747
Running dot for graph 246/747
Running dot for graph 247/747
Running dot for graph 248/747
Running dot for graph 249/747
Running dot for graph 250/747
Running dot for graph 251/747
Running dot for graph 252/747
Running dot for graph 253/747
Running dot for graph 254/747
Running dot for graph 255/747
Running dot for graph 256/747
Running dot for graph 257/747
Running dot for graph 258/747
Running dot for graph 259/747
Running dot for graph 260/747
Running dot for graph 261/747
Running dot for graph 262/747
Running dot for graph 263/747
Running dot for graph 264/747
Running dot for graph 265/747
Running dot for graph 266/747
Running dot for graph 267/747
Running dot for graph 268/747
Running dot for graph 269/747
Running dot for graph 270/747
Running dot for graph 271/747
Running dot for graph 272/747
Running dot for graph 273/747
Running dot for graph 274/747
Running dot for graph 275/747
Running dot for graph 276/747
Running dot for graph 277/747
Running dot for graph 278/747
Running dot for graph 279/747
Running dot for graph 280/747
Running dot for graph 281/747
Running dot for graph 282/747
Running dot for graph 283/747
Running dot for graph 284/747
Running dot for graph 285/747
Running dot for graph 286/747
Running dot for graph 287/747
Running dot for graph 288/747
Running dot for graph 289/747
Running dot for graph 290/747
Running dot for graph 291/747
Running dot for graph 292/747
Running dot for graph 293/747
Running dot for graph 294/747
Running dot for graph 295/747
Running dot for graph 296/747
Running dot for graph 297/747
Running dot for graph 298/747
Running dot for graph 299/747
Running dot for graph 300/747
Running dot for graph 301/747
Running dot for graph 302/747
Running dot for graph 303/747
Running dot for graph 304/747
Running dot for graph 305/747
Running dot for graph 306/747
Running dot for graph 307/747
Running dot for graph 308/747
Running dot for graph 309/747
Running dot for graph 310/747
Running dot for graph 311/747
Running dot for graph 312/747
Running dot for graph 313/747
Running dot for graph 314/747
Running dot for graph 315/747
Running dot for graph 316/747
Running dot for graph 317/747
Running dot for graph 318/747
Running dot for graph 319/747
Running dot for graph 320/747
Running dot for graph 321/747
Running dot for graph 322/747
Running dot for graph 323/747
Running dot for graph 324/747
Running dot for graph 325/747
Running dot for graph 326/747
Running dot for graph 327/747
Running dot for graph 328/747
Running dot for graph 329/747
Running dot for graph 330/747
Running dot for graph 331/747
Running dot for graph 332/747
Running dot for graph 333/747
Running dot for graph 334/747
Running dot for graph 335/747
Running dot for graph 336/747
Running dot for graph 337/747
Running dot for graph 338/747
Running dot for graph 339/747
Running dot for graph 340/747
Running dot for graph 341/747
Running dot for graph 342/747
Running dot for graph 343/747
Running dot for graph 344/747
Running dot for graph 345/747
Running dot for graph 346/747
Running dot for graph 347/747
Running dot for graph 348/747
Running dot for graph 349/747
Running dot for graph 350/747
Running dot for graph 351/747
Running dot for graph 352/747
Running dot for graph 353/747
Running dot for graph 354/747
Running dot for graph 355/747
Running dot for graph 356/747
Running dot for graph 357/747
Running dot for graph 358/747
Running dot for graph 359/747
Running dot for graph 360/747
Running dot for graph 361/747
Running dot for graph 362/747
Running dot for graph 363/747
Running dot for graph 364/747
Running dot for graph 365/747
Running dot for graph 366/747
Running dot for graph 367/747
Running dot for graph 368/747
Running dot for graph 369/747
Running dot for graph 370/747
Running dot for graph 371/747
Running dot for graph 372/747
Running dot for graph 373/747
Running dot for graph 374/747
Running dot for graph 375/747
Running dot for graph 376/747
Running dot for graph 377/747
Running dot for graph 378/747
Running dot for graph 379/747
Running dot for graph 380/747
Running dot for graph 381/747
Running dot for graph 382/747
Running dot for graph 383/747
Running dot for graph 384/747
Running dot for graph 385/747
Running dot for graph 386/747
Running dot for graph 387/747
Running dot for graph 388/747
Running dot for graph 389/747
Running dot for graph 390/747
Running dot for graph 391/747
Running dot for graph 392/747
Running dot for graph 393/747
Running dot for graph 394/747
Running dot for graph 395/747
Running dot for graph 396/747
Running dot for graph 397/747
Running dot for graph 398/747
Running dot for graph 399/747
Running dot for graph 400/747
Running dot for graph 401/747
Running dot for graph 402/747
Running dot for graph 403/747
Running dot for graph 404/747
Running dot for graph 405/747
Running dot for graph 406/747
Running dot for graph 407/747
Running dot for graph 408/747
Running dot for graph 409/747
Running dot for graph 410/747
Running dot for graph 411/747
Running dot for graph 412/747
Running dot for graph 413/747
Running dot for graph 414/747
Running dot for graph 415/747
Running dot for graph 416/747
Running dot for graph 417/747
Running dot for graph 418/747
Running dot for graph 419/747
Running dot for graph 420/747
Running dot for graph 421/747
Running dot for graph 422/747
Running dot for graph 423/747
Running dot for graph 424/747
Running dot for graph 425/747
Running dot for graph 426/747
Running dot for graph 427/747
Running dot for graph 428/747
Running dot for graph 429/747
Running dot for graph 430/747
Running dot for graph 431/747
Running dot for graph 432/747
Running dot for graph 433/747
Running dot for graph 434/747
Running dot for graph 435/747
Running dot for graph 436/747
Running dot for graph 437/747
Running dot for graph 438/747
Running dot for graph 439/747
Running dot for graph 440/747
Running dot for graph 441/747
Running dot for graph 442/747
Running dot for graph 443/747
Running dot for graph 444/747
Running dot for graph 445/747
Running dot for graph 446/747
Running dot for graph 447/747
Running dot for graph 448/747
Running dot for graph 449/747
Running dot for graph 450/747
Running dot for graph 451/747
Running dot for graph 452/747
Running dot for graph 453/747
Running dot for graph 454/747
Running dot for graph 455/747
Running dot for graph 456/747
Running dot for graph 457/747
Running dot for graph 458/747
Running dot for graph 459/747
Running dot for graph 460/747
Running dot for graph 461/747
Running dot for graph 462/747
Running dot for graph 463/747
Running dot for graph 464/747
Running dot for graph 465/747
Running dot for graph 466/747
Running dot for graph 467/747
Running dot for graph 468/747
Running dot for graph 469/747
Running dot for graph 470/747
Running dot for graph 471/747
Running dot for graph 472/747
Running dot for graph 473/747
Running dot for graph 474/747
Running dot for graph 475/747
Running dot for graph 476/747
Running dot for graph 477/747
Running dot for graph 478/747
Running dot for graph 479/747
Running dot for graph 480/747
Running dot for graph 481/747
Running dot for graph 482/747
Running dot for graph 483/747
Running dot for graph 484/747
Running dot for graph 485/747
Running dot for graph 486/747
Running dot for graph 487/747
Running dot for graph 488/747
Running dot for graph 489/747
Running dot for graph 490/747
Running dot for graph 491/747
Running dot for graph 492/747
Running dot for graph 493/747
Running dot for graph 494/747
Running dot for graph 495/747
Running dot for graph 496/747
Running dot for graph 497/747
Running dot for graph 498/747
Running dot for graph 499/747
Running dot for graph 500/747
Running dot for graph 501/747
Running dot for graph 502/747
Running dot for graph 503/747
Running dot for graph 504/747
Running dot for graph 505/747
Running dot for graph 506/747
Running dot for graph 507/747
Running dot for graph 508/747
Running dot for graph 509/747
Running dot for graph 510/747
Running dot for graph 511/747
Running dot for graph 512/747
Running dot for graph 513/747
Running dot for graph 514/747
Running dot for graph 515/747
Running dot for graph 516/747
Running dot for graph 517/747
Running dot for graph 518/747
Running dot for graph 519/747
Running dot for graph 520/747
Running dot for graph 521/747
Running dot for graph 522/747
Running dot for graph 523/747
Running dot for graph 524/747
Running dot for graph 525/747
Running dot for graph 526/747
Running dot for graph 527/747
Running dot for graph 528/747
Running dot for graph 529/747
Running dot for graph 530/747
Running dot for graph 531/747
Running dot for graph 532/747
Running dot for graph 533/747
Running dot for graph 534/747
Running dot for graph 535/747
Running dot for graph 536/747
Running dot for graph 537/747
Running dot for graph 538/747
Running dot for graph 539/747
Running dot for graph 540/747
Running dot for graph 541/747
Running dot for graph 542/747
Running dot for graph 543/747
Running dot for graph 544/747
Running dot for graph 545/747
Running dot for graph 546/747
Running dot for graph 547/747
Running dot for graph 548/747
Running dot for graph 549/747
Running dot for graph 550/747
Running dot for graph 551/747
Running dot for graph 552/747
Running dot for graph 553/747
Running dot for graph 554/747
Running dot for graph 555/747
Running dot for graph 556/747
Running dot for graph 557/747
Running dot for graph 558/747
Running dot for graph 559/747
Running dot for graph 560/747
Running dot for graph 561/747
Running dot for graph 562/747
Running dot for graph 563/747
Running dot for graph 564/747
Running dot for graph 565/747
Running dot for graph 566/747
Running dot for graph 567/747
Running dot for graph 568/747
Running dot for graph 569/747
Running dot for graph 570/747
Running dot for graph 571/747
Running dot for graph 572/747
Running dot for graph 573/747
Running dot for graph 574/747
Running dot for graph 575/747
Running dot for graph 576/747
Running dot for graph 577/747
Running dot for graph 578/747
Running dot for graph 579/747
Running dot for graph 580/747
Running dot for graph 581/747
Running dot for graph 582/747
Running dot for graph 583/747
Running dot for graph 584/747
Running dot for graph 585/747
Running dot for graph 586/747
Running dot for graph 587/747
Running dot for graph 588/747
Running dot for graph 589/747
Running dot for graph 590/747
Running dot for graph 591/747
Running dot for graph 592/747
Running dot for graph 593/747
Running dot for graph 594/747
Running dot for graph 595/747
Running dot for graph 596/747
Running dot for graph 597/747
Running dot for graph 598/747
Running dot for graph 599/747
Running dot for graph 600/747
Running dot for graph 601/747
Running dot for graph 602/747
Running dot for graph 603/747
Running dot for graph 604/747
Running dot for graph 605/747
Running dot for graph 606/747
Running dot for graph 607/747
Running dot for graph 608/747
Running dot for graph 609/747
Running dot for graph 610/747
Running dot for graph 611/747
Running dot for graph 612/747
Running dot for graph 613/747
Running dot for graph 614/747
Running dot for graph 615/747
Running dot for graph 616/747
Running dot for graph 617/747
Running dot for graph 618/747
Running dot for graph 619/747
Running dot for graph 620/747
Running dot for graph 621/747
Running dot for graph 622/747
Running dot for graph 623/747
Running dot for graph 624/747
Running dot for graph 625/747
Running dot for graph 626/747
Running dot for graph 627/747
Running dot for graph 628/747
Running dot for graph 629/747
Running dot for graph 630/747
Running dot for graph 631/747
Running dot for graph 632/747
Running dot for graph 633/747
Running dot for graph 634/747
Running dot for graph 635/747
Running dot for graph 636/747
Running dot for graph 637/747
Running dot for graph 638/747
Running dot for graph 639/747
Running dot for graph 640/747
Running dot for graph 641/747
Running dot for graph 642/747
Running dot for graph 643/747
Running dot for graph 644/747
Running dot for graph 645/747
Running dot for graph 646/747
Running dot for graph 647/747
Running dot for graph 648/747
Running dot for graph 649/747
Running dot for graph 650/747
Running dot for graph 651/747
Running dot for graph 652/747
Running dot for graph 653/747
Running dot for graph 654/747
Running dot for graph 655/747
Running dot for graph 656/747
Running dot for graph 657/747
Running dot for graph 658/747
Running dot for graph 659/747
Running dot for graph 660/747
Running dot for graph 661/747
Running dot for graph 662/747
Running dot for graph 663/747
Running dot for graph 664/747
Running dot for graph 665/747
Running dot for graph 666/747
Running dot for graph 667/747
Running dot for graph 668/747
Running dot for graph 669/747
Running dot for graph 670/747
Running dot for graph 671/747
Running dot for graph 672/747
Running dot for graph 673/747
Running dot for graph 674/747
Running dot for graph 675/747
Running dot for graph 676/747
Running dot for graph 677/747
Running dot for graph 678/747
Running dot for graph 679/747
Running dot for graph 680/747
Running dot for graph 681/747
Running dot for graph 682/747
Running dot for graph 683/747
Running dot for graph 684/747
Running dot for graph 685/747
Running dot for graph 686/747
Running dot for graph 687/747
Running dot for graph 688/747
Running dot for graph 689/747
Running dot for graph 690/747
Running dot for graph 691/747
Running dot for graph 692/747
Running dot for graph 693/747
Running dot for graph 694/747
Running dot for graph 695/747
Running dot for graph 696/747
Running dot for graph 697/747
Running dot for graph 698/747
Running dot for graph 699/747
Running dot for graph 700/747
Running dot for graph 701/747
Running dot for graph 702/747
Running dot for graph 703/747
Running dot for graph 704/747
Running dot for graph 705/747
Running dot for graph 706/747
Running dot for graph 707/747
Running dot for graph 708/747
Running dot for graph 709/747
Running dot for graph 710/747
Running dot for graph 711/747
Running dot for graph 712/747
Running dot for graph 713/747
Running dot for graph 714/747
Running dot for graph 715/747
Running dot for graph 716/747
Running dot for graph 717/747
Running dot for graph 718/747
Running dot for graph 719/747
Running dot for graph 720/747
Running dot for graph 721/747
Running dot for graph 722/747
Running dot for graph 723/747
Running dot for graph 724/747
Running dot for graph 725/747
Running dot for graph 726/747
Running dot for graph 727/747
Running dot for graph 728/747
Running dot for graph 729/747
Running dot for graph 730/747
Running dot for graph 731/747
Running dot for graph 732/747
Running dot for graph 733/747
Running dot for graph 734/747
Running dot for graph 735/747
Running dot for graph 736/747
Running dot for graph 737/747
Running dot for graph 738/747
Running dot for graph 739/747
Running dot for graph 740/747
Running dot for graph 741/747
Running dot for graph 742/747
Running dot for graph 743/747
Running dot for graph 744/747
Running dot for graph 745/747
Running dot for graph 746/747
Running dot for graph 747/747
Patching output file 1/1052
Patching output file 2/1052
Patching output file 3/1052
Patching output file 4/1052
Patching output file 5/1052
Patching output file 6/1052
Patching output file 7/1052
Patching output file 8/1052
Patching output file 9/1052
Patching output file 10/1052
Patching output file 11/1052
Patching output file 12/1052
Patching output file 13/1052
Patching output file 14/1052
Patching output file 15/1052
Patching output file 16/1052
Patching output file 17/1052
Patching output file 18/1052
Patching output file 19/1052
Patching output file 20/1052
Patching output file 21/1052
Patching output file 22/1052
Patching output file 23/1052
Patching output file 24/1052
Patching output file 25/1052
Patching output file 26/1052
Patching output file 27/1052
Patching output file 28/1052
Patching output file 29/1052
Patching output file 30/1052
Patching output file 31/1052
Patching output file 32/1052
Patching output file 33/1052
Patching output file 34/1052
Patching output file 35/1052
Patching output file 36/1052
Patching output file 37/1052
Patching output file 38/1052
Patching output file 39/1052
Patching output file 40/1052
Patching output file 41/1052
Patching output file 42/1052
Patching output file 43/1052
Patching output file 44/1052
Patching output file 45/1052
Patching output file 46/1052
Patching output file 47/1052
Patching output file 48/1052
Patching output file 49/1052
Patching output file 50/1052
Patching output file 51/1052
Patching output file 52/1052
Patching output file 53/1052
Patching output file 54/1052
Patching output file 55/1052
Patching output file 56/1052
Patching output file 57/1052
Patching output file 58/1052
Patching output file 59/1052
Patching output file 60/1052
Patching output file 61/1052
Patching output file 62/1052
Patching output file 63/1052
Patching output file 64/1052
Patching output file 65/1052
Patching output file 66/1052
Patching output file 67/1052
Patching output file 68/1052
Patching output file 69/1052
Patching output file 70/1052
Patching output file 71/1052
Patching output file 72/1052
Patching output file 73/1052
Patching output file 74/1052
Patching output file 75/1052
Patching output file 76/1052
Patching output file 77/1052
Patching output file 78/1052
Patching output file 79/1052
Patching output file 80/1052
Patching output file 81/1052
Patching output file 82/1052
Patching output file 83/1052
Patching output file 84/1052
Patching output file 85/1052
Patching output file 86/1052
Patching output file 87/1052
Patching output file 88/1052
Patching output file 89/1052
Patching output file 90/1052
Patching output file 91/1052
Patching output file 92/1052
Patching output file 93/1052
Patching output file 94/1052
Patching output file 95/1052
Patching output file 96/1052
Patching output file 97/1052
Patching output file 98/1052
Patching output file 99/1052
Patching output file 100/1052
Patching output file 101/1052
Patching output file 102/1052
Patching output file 103/1052
Patching output file 104/1052
Patching output file 105/1052
Patching output file 106/1052
Patching output file 107/1052
Patching output file 108/1052
Patching output file 109/1052
Patching output file 110/1052
Patching output file 111/1052
Patching output file 112/1052
Patching output file 113/1052
Patching output file 114/1052
Patching output file 115/1052
Patching output file 116/1052
Patching output file 117/1052
Patching output file 118/1052
Patching output file 119/1052
Patching output file 120/1052
Patching output file 121/1052
Patching output file 122/1052
Patching output file 123/1052
Patching output file 124/1052
Patching output file 125/1052
Patching output file 126/1052
Patching output file 127/1052
Patching output file 128/1052
Patching output file 129/1052
Patching output file 130/1052
Patching output file 131/1052
Patching output file 132/1052
Patching output file 133/1052
Patching output file 134/1052
Patching output file 135/1052
Patching output file 136/1052
Patching output file 137/1052
Patching output file 138/1052
Patching output file 139/1052
Patching output file 140/1052
Patching output file 141/1052
Patching output file 142/1052
Patching output file 143/1052
Patching output file 144/1052
Patching output file 145/1052
Patching output file 146/1052
Patching output file 147/1052
Patching output file 148/1052
Patching output file 149/1052
Patching output file 150/1052
Patching output file 151/1052
Patching output file 152/1052
Patching output file 153/1052
Patching output file 154/1052
Patching output file 155/1052
Patching output file 156/1052
Patching output file 157/1052
Patching output file 158/1052
Patching output file 159/1052
Patching output file 160/1052
Patching output file 161/1052
Patching output file 162/1052
Patching output file 163/1052
Patching output file 164/1052
Patching output file 165/1052
Patching output file 166/1052
Patching output file 167/1052
Patching output file 168/1052
Patching output file 169/1052
Patching output file 170/1052
Patching output file 171/1052
Patching output file 172/1052
Patching output file 173/1052
Patching output file 174/1052
Patching output file 175/1052
Patching output file 176/1052
Patching output file 177/1052
Patching output file 178/1052
Patching output file 179/1052
Patching output file 180/1052
Patching output file 181/1052
Patching output file 182/1052
Patching output file 183/1052
Patching output file 184/1052
Patching output file 185/1052
Patching output file 186/1052
Patching output file 187/1052
Patching output file 188/1052
Patching output file 189/1052
Patching output file 190/1052
Patching output file 191/1052
Patching output file 192/1052
Patching output file 193/1052
Patching output file 194/1052
Patching output file 195/1052
Patching output file 196/1052
Patching output file 197/1052
Patching output file 198/1052
Patching output file 199/1052
Patching output file 200/1052
Patching output file 201/1052
Patching output file 202/1052
Patching output file 203/1052
Patching output file 204/1052
Patching output file 205/1052
Patching output file 206/1052
Patching output file 207/1052
Patching output file 208/1052
Patching output file 209/1052
Patching output file 210/1052
Patching output file 211/1052
Patching output file 212/1052
Patching output file 213/1052
Patching output file 214/1052
Patching output file 215/1052
Patching output file 216/1052
Patching output file 217/1052
Patching output file 218/1052
Patching output file 219/1052
Patching output file 220/1052
Patching output file 221/1052
Patching output file 222/1052
Patching output file 223/1052
Patching output file 224/1052
Patching output file 225/1052
Patching output file 226/1052
Patching output file 227/1052
Patching output file 228/1052
Patching output file 229/1052
Patching output file 230/1052
Patching output file 231/1052
Patching output file 232/1052
Patching output file 233/1052
Patching output file 234/1052
Patching output file 235/1052
Patching output file 236/1052
Patching output file 237/1052
Patching output file 238/1052
Patching output file 239/1052
Patching output file 240/1052
Patching output file 241/1052
Patching output file 242/1052
Patching output file 243/1052
Patching output file 244/1052
Patching output file 245/1052
Patching output file 246/1052
Patching output file 247/1052
Patching output file 248/1052
Patching output file 249/1052
Patching output file 250/1052
Patching output file 251/1052
Patching output file 252/1052
Patching output file 253/1052
Patching output file 254/1052
Patching output file 255/1052
Patching output file 256/1052
Patching output file 257/1052
Patching output file 258/1052
Patching output file 259/1052
Patching output file 260/1052
Patching output file 261/1052
Patching output file 262/1052
Patching output file 263/1052
Patching output file 264/1052
Patching output file 265/1052
Patching output file 266/1052
Patching output file 267/1052
Patching output file 268/1052
Patching output file 269/1052
Patching output file 270/1052
Patching output file 271/1052
Patching output file 272/1052
Patching output file 273/1052
Patching output file 274/1052
Patching output file 275/1052
Patching output file 276/1052
Patching output file 277/1052
Patching output file 278/1052
Patching output file 279/1052
Patching output file 280/1052
Patching output file 281/1052
Patching output file 282/1052
Patching output file 283/1052
Patching output file 284/1052
Patching output file 285/1052
Patching output file 286/1052
Patching output file 287/1052
Patching output file 288/1052
Patching output file 289/1052
Patching output file 290/1052
Patching output file 291/1052
Patching output file 292/1052
Patching output file 293/1052
Patching output file 294/1052
Patching output file 295/1052
Patching output file 296/1052
Patching output file 297/1052
Patching output file 298/1052
Patching output file 299/1052
Patching output file 300/1052
Patching output file 301/1052
Patching output file 302/1052
Patching output file 303/1052
Patching output file 304/1052
Patching output file 305/1052
Patching output file 306/1052
Patching output file 307/1052
Patching output file 308/1052
Patching output file 309/1052
Patching output file 310/1052
Patching output file 311/1052
Patching output file 312/1052
Patching output file 313/1052
Patching output file 314/1052
Patching output file 315/1052
Patching output file 316/1052
Patching output file 317/1052
Patching output file 318/1052
Patching output file 319/1052
Patching output file 320/1052
Patching output file 321/1052
Patching output file 322/1052
Patching output file 323/1052
Patching output file 324/1052
Patching output file 325/1052
Patching output file 326/1052
Patching output file 327/1052
Patching output file 328/1052
Patching output file 329/1052
Patching output file 330/1052
Patching output file 331/1052
Patching output file 332/1052
Patching output file 333/1052
Patching output file 334/1052
Patching output file 335/1052
Patching output file 336/1052
Patching output file 337/1052
Patching output file 338/1052
Patching output file 339/1052
Patching output file 340/1052
Patching output file 341/1052
Patching output file 342/1052
Patching output file 343/1052
Patching output file 344/1052
Patching output file 345/1052
Patching output file 346/1052
Patching output file 347/1052
Patching output file 348/1052
Patching output file 349/1052
Patching output file 350/1052
Patching output file 351/1052
Patching output file 352/1052
Patching output file 353/1052
Patching output file 354/1052
Patching output file 355/1052
Patching output file 356/1052
Patching output file 357/1052
Patching output file 358/1052
Patching output file 359/1052
Patching output file 360/1052
Patching output file 361/1052
Patching output file 362/1052
Patching output file 363/1052
Patching output file 364/1052
Patching output file 365/1052
Patching output file 366/1052
Patching output file 367/1052
Patching output file 368/1052
Patching output file 369/1052
Patching output file 370/1052
Patching output file 371/1052
Patching output file 372/1052
Patching output file 373/1052
Patching output file 374/1052
Patching output file 375/1052
Patching output file 376/1052
Patching output file 377/1052
Patching output file 378/1052
Patching output file 379/1052
Patching output file 380/1052
Patching output file 381/1052
Patching output file 382/1052
Patching output file 383/1052
Patching output file 384/1052
Patching output file 385/1052
Patching output file 386/1052
Patching output file 387/1052
Patching output file 388/1052
Patching output file 389/1052
Patching output file 390/1052
Patching output file 391/1052
Patching output file 392/1052
Patching output file 393/1052
Patching output file 394/1052
Patching output file 395/1052
Patching output file 396/1052
Patching output file 397/1052
Patching output file 398/1052
Patching output file 399/1052
Patching output file 400/1052
Patching output file 401/1052
Patching output file 402/1052
Patching output file 403/1052
Patching output file 404/1052
Patching output file 405/1052
Patching output file 406/1052
Patching output file 407/1052
Patching output file 408/1052
Patching output file 409/1052
Patching output file 410/1052
Patching output file 411/1052
Patching output file 412/1052
Patching output file 413/1052
Patching output file 414/1052
Patching output file 415/1052
Patching output file 416/1052
Patching output file 417/1052
Patching output file 418/1052
Patching output file 419/1052
Patching output file 420/1052
Patching output file 421/1052
Patching output file 422/1052
Patching output file 423/1052
Patching output file 424/1052
Patching output file 425/1052
Patching output file 426/1052
Patching output file 427/1052
Patching output file 428/1052
Patching output file 429/1052
Patching output file 430/1052
Patching output file 431/1052
Patching output file 432/1052
Patching output file 433/1052
Patching output file 434/1052
Patching output file 435/1052
Patching output file 436/1052
Patching output file 437/1052
Patching output file 438/1052
Patching output file 439/1052
Patching output file 440/1052
Patching output file 441/1052
Patching output file 442/1052
Patching output file 443/1052
Patching output file 444/1052
Patching output file 445/1052
Patching output file 446/1052
Patching output file 447/1052
Patching output file 448/1052
Patching output file 449/1052
Patching output file 450/1052
Patching output file 451/1052
Patching output file 452/1052
Patching output file 453/1052
Patching output file 454/1052
Patching output file 455/1052
Patching output file 456/1052
Patching output file 457/1052
Patching output file 458/1052
Patching output file 459/1052
Patching output file 460/1052
Patching output file 461/1052
Patching output file 462/1052
Patching output file 463/1052
Patching output file 464/1052
Patching output file 465/1052
Patching output file 466/1052
Patching output file 467/1052
Patching output file 468/1052
Patching output file 469/1052
Patching output file 470/1052
Patching output file 471/1052
Patching output file 472/1052
Patching output file 473/1052
Patching output file 474/1052
Patching output file 475/1052
Patching output file 476/1052
Patching output file 477/1052
Patching output file 478/1052
Patching output file 479/1052
Patching output file 480/1052
Patching output file 481/1052
Patching output file 482/1052
Patching output file 483/1052
Patching output file 484/1052
Patching output file 485/1052
Patching output file 486/1052
Patching output file 487/1052
Patching output file 488/1052
Patching output file 489/1052
Patching output file 490/1052
Patching output file 491/1052
Patching output file 492/1052
Patching output file 493/1052
Patching output file 494/1052
Patching output file 495/1052
Patching output file 496/1052
Patching output file 497/1052
Patching output file 498/1052
Patching output file 499/1052
Patching output file 500/1052
Patching output file 501/1052
Patching output file 502/1052
Patching output file 503/1052
Patching output file 504/1052
Patching output file 505/1052
Patching output file 506/1052
Patching output file 507/1052
Patching output file 508/1052
Patching output file 509/1052
Patching output file 510/1052
Patching output file 511/1052
Patching output file 512/1052
Patching output file 513/1052
Patching output file 514/1052
Patching output file 515/1052
Patching output file 516/1052
Patching output file 517/1052
Patching output file 518/1052
Patching output file 519/1052
Patching output file 520/1052
Patching output file 521/1052
Patching output file 522/1052
Patching output file 523/1052
Patching output file 524/1052
Patching output file 525/1052
Patching output file 526/1052
Patching output file 527/1052
Patching output file 528/1052
Patching output file 529/1052
Patching output file 530/1052
Patching output file 531/1052
Patching output file 532/1052
Patching output file 533/1052
Patching output file 534/1052
Patching output file 535/1052
Patching output file 536/1052
Patching output file 537/1052
Patching output file 538/1052
Patching output file 539/1052
Patching output file 540/1052
Patching output file 541/1052
Patching output file 542/1052
Patching output file 543/1052
Patching output file 544/1052
Patching output file 545/1052
Patching output file 546/1052
Patching output file 547/1052
Patching output file 548/1052
Patching output file 549/1052
Patching output file 550/1052
Patching output file 551/1052
Patching output file 552/1052
Patching output file 553/1052
Patching output file 554/1052
Patching output file 555/1052
Patching output file 556/1052
Patching output file 557/1052
Patching output file 558/1052
Patching output file 559/1052
Patching output file 560/1052
Patching output file 561/1052
Patching output file 562/1052
Patching output file 563/1052
Patching output file 564/1052
Patching output file 565/1052
Patching output file 566/1052
Patching output file 567/1052
Patching output file 568/1052
Patching output file 569/1052
Patching output file 570/1052
Patching output file 571/1052
Patching output file 572/1052
Patching output file 573/1052
Patching output file 574/1052
Patching output file 575/1052
Patching output file 576/1052
Patching output file 577/1052
Patching output file 578/1052
Patching output file 579/1052
Patching output file 580/1052
Patching output file 581/1052
Patching output file 582/1052
Patching output file 583/1052
Patching output file 584/1052
Patching output file 585/1052
Patching output file 586/1052
Patching output file 587/1052
Patching output file 588/1052
Patching output file 589/1052
Patching output file 590/1052
Patching output file 591/1052
Patching output file 592/1052
Patching output file 593/1052
Patching output file 594/1052
Patching output file 595/1052
Patching output file 596/1052
Patching output file 597/1052
Patching output file 598/1052
Patching output file 599/1052
Patching output file 600/1052
Patching output file 601/1052
Patching output file 602/1052
Patching output file 603/1052
Patching output file 604/1052
Patching output file 605/1052
Patching output file 606/1052
Patching output file 607/1052
Patching output file 608/1052
Patching output file 609/1052
Patching output file 610/1052
Patching output file 611/1052
Patching output file 612/1052
Patching output file 613/1052
Patching output file 614/1052
Patching output file 615/1052
Patching output file 616/1052
Patching output file 617/1052
Patching output file 618/1052
Patching output file 619/1052
Patching output file 620/1052
Patching output file 621/1052
Patching output file 622/1052
Patching output file 623/1052
Patching output file 624/1052
Patching output file 625/1052
Patching output file 626/1052
Patching output file 627/1052
Patching output file 628/1052
Patching output file 629/1052
Patching output file 630/1052
Patching output file 631/1052
Patching output file 632/1052
Patching output file 633/1052
Patching output file 634/1052
Patching output file 635/1052
Patching output file 636/1052
Patching output file 637/1052
Patching output file 638/1052
Patching output file 639/1052
Patching output file 640/1052
Patching output file 641/1052
Patching output file 642/1052
Patching output file 643/1052
Patching output file 644/1052
Patching output file 645/1052
Patching output file 646/1052
Patching output file 647/1052
Patching output file 648/1052
Patching output file 649/1052
Patching output file 650/1052
Patching output file 651/1052
Patching output file 652/1052
Patching output file 653/1052
Patching output file 654/1052
Patching output file 655/1052
Patching output file 656/1052
Patching output file 657/1052
Patching output file 658/1052
Patching output file 659/1052
Patching output file 660/1052
Patching output file 661/1052
Patching output file 662/1052
Patching output file 663/1052
Patching output file 664/1052
Patching output file 665/1052
Patching output file 666/1052
Patching output file 667/1052
Patching output file 668/1052
Patching output file 669/1052
Patching output file 670/1052
Patching output file 671/1052
Patching output file 672/1052
Patching output file 673/1052
Patching output file 674/1052
Patching output file 675/1052
Patching output file 676/1052
Patching output file 677/1052
Patching output file 678/1052
Patching output file 679/1052
Patching output file 680/1052
Patching output file 681/1052
Patching output file 682/1052
Patching output file 683/1052
Patching output file 684/1052
Patching output file 685/1052
Patching output file 686/1052
Patching output file 687/1052
Patching output file 688/1052
Patching output file 689/1052
Patching output file 690/1052
Patching output file 691/1052
Patching output file 692/1052
Patching output file 693/1052
Patching output file 694/1052
Patching output file 695/1052
Patching output file 696/1052
Patching output file 697/1052
Patching output file 698/1052
Patching output file 699/1052
Patching output file 700/1052
Patching output file 701/1052
Patching output file 702/1052
Patching output file 703/1052
Patching output file 704/1052
Patching output file 705/1052
Patching output file 706/1052
Patching output file 707/1052
Patching output file 708/1052
Patching output file 709/1052
Patching output file 710/1052
Patching output file 711/1052
Patching output file 712/1052
Patching output file 713/1052
Patching output file 714/1052
Patching output file 715/1052
Patching output file 716/1052
Patching output file 717/1052
Patching output file 718/1052
Patching output file 719/1052
Patching output file 720/1052
Patching output file 721/1052
Patching output file 722/1052
Patching output file 723/1052
Patching output file 724/1052
Patching output file 725/1052
Patching output file 726/1052
Patching output file 727/1052
Patching output file 728/1052
Patching output file 729/1052
Patching output file 730/1052
Patching output file 731/1052
Patching output file 732/1052
Patching output file 733/1052
Patching output file 734/1052
Patching output file 735/1052
Patching output file 736/1052
Patching output file 737/1052
Patching output file 738/1052
Patching output file 739/1052
Patching output file 740/1052
Patching output file 741/1052
Patching output file 742/1052
Patching output file 743/1052
Patching output file 744/1052
Patching output file 745/1052
Patching output file 746/1052
Patching output file 747/1052
Patching output file 748/1052
Patching output file 749/1052
Patching output file 750/1052
Patching output file 751/1052
Patching output file 752/1052
Patching output file 753/1052
Patching output file 754/1052
Patching output file 755/1052
Patching output file 756/1052
Patching output file 757/1052
Patching output file 758/1052
Patching output file 759/1052
Patching output file 760/1052
Patching output file 761/1052
Patching output file 762/1052
Patching output file 763/1052
Patching output file 764/1052
Patching output file 765/1052
Patching output file 766/1052
Patching output file 767/1052
Patching output file 768/1052
Patching output file 769/1052
Patching output file 770/1052
Patching output file 771/1052
Patching output file 772/1052
Patching output file 773/1052
Patching output file 774/1052
Patching output file 775/1052
Patching output file 776/1052
Patching output file 777/1052
Patching output file 778/1052
Patching output file 779/1052
Patching output file 780/1052
Patching output file 781/1052
Patching output file 782/1052
Patching output file 783/1052
Patching output file 784/1052
Patching output file 785/1052
Patching output file 786/1052
Patching output file 787/1052
Patching output file 788/1052
Patching output file 789/1052
Patching output file 790/1052
Patching output file 791/1052
Patching output file 792/1052
Patching output file 793/1052
Patching output file 794/1052
Patching output file 795/1052
Patching output file 796/1052
Patching output file 797/1052
Patching output file 798/1052
Patching output file 799/1052
Patching output file 800/1052
Patching output file 801/1052
Patching output file 802/1052
Patching output file 803/1052
Patching output file 804/1052
Patching output file 805/1052
Patching output file 806/1052
Patching output file 807/1052
Patching output file 808/1052
Patching output file 809/1052
Patching output file 810/1052
Patching output file 811/1052
Patching output file 812/1052
Patching output file 813/1052
Patching output file 814/1052
Patching output file 815/1052
Patching output file 816/1052
Patching output file 817/1052
Patching output file 818/1052
Patching output file 819/1052
Patching output file 820/1052
Patching output file 821/1052
Patching output file 822/1052
Patching output file 823/1052
Patching output file 824/1052
Patching output file 825/1052
Patching output file 826/1052
Patching output file 827/1052
Patching output file 828/1052
Patching output file 829/1052
Patching output file 830/1052
Patching output file 831/1052
Patching output file 832/1052
Patching output file 833/1052
Patching output file 834/1052
Patching output file 835/1052
Patching output file 836/1052
Patching output file 837/1052
Patching output file 838/1052
Patching output file 839/1052
Patching output file 840/1052
Patching output file 841/1052
Patching output file 842/1052
Patching output file 843/1052
Patching output file 844/1052
Patching output file 845/1052
Patching output file 846/1052
Patching output file 847/1052
Patching output file 848/1052
Patching output file 849/1052
Patching output file 850/1052
Patching output file 851/1052
Patching output file 852/1052
Patching output file 853/1052
Patching output file 854/1052
Patching output file 855/1052
Patching output file 856/1052
Patching output file 857/1052
Patching output file 858/1052
Patching output file 859/1052
Patching output file 860/1052
Patching output file 861/1052
Patching output file 862/1052
Patching output file 863/1052
Patching output file 864/1052
Patching output file 865/1052
Patching output file 866/1052
Patching output file 867/1052
Patching output file 868/1052
Patching output file 869/1052
Patching output file 870/1052
Patching output file 871/1052
Patching output file 872/1052
Patching output file 873/1052
Patching output file 874/1052
Patching output file 875/1052
Patching output file 876/1052
Patching output file 877/1052
Patching output file 878/1052
Patching output file 879/1052
Patching output file 880/1052
Patching output file 881/1052
Patching output file 882/1052
Patching output file 883/1052
Patching output file 884/1052
Patching output file 885/1052
Patching output file 886/1052
Patching output file 887/1052
Patching output file 888/1052
Patching output file 889/1052
Patching output file 890/1052
Patching output file 891/1052
Patching output file 892/1052
Patching output file 893/1052
Patching output file 894/1052
Patching output file 895/1052
Patching output file 896/1052
Patching output file 897/1052
Patching output file 898/1052
Patching output file 899/1052
Patching output file 900/1052
Patching output file 901/1052
Patching output file 902/1052
Patching output file 903/1052
Patching output file 904/1052
Patching output file 905/1052
Patching output file 906/1052
Patching output file 907/1052
Patching output file 908/1052
Patching output file 909/1052
Patching output file 910/1052
Patching output file 911/1052
Patching output file 912/1052
Patching output file 913/1052
Patching output file 914/1052
Patching output file 915/1052
Patching output file 916/1052
Patching output file 917/1052
Patching output file 918/1052
Patching output file 919/1052
Patching output file 920/1052
Patching output file 921/1052
Patching output file 922/1052
Patching output file 923/1052
Patching output file 924/1052
Patching output file 925/1052
Patching output file 926/1052
Patching output file 927/1052
Patching output file 928/1052
Patching output file 929/1052
Patching output file 930/1052
Patching output file 931/1052
Patching output file 932/1052
Patching output file 933/1052
Patching output file 934/1052
Patching output file 935/1052
Patching output file 936/1052
Patching output file 937/1052
Patching output file 938/1052
Patching output file 939/1052
Patching output file 940/1052
Patching output file 941/1052
Patching output file 942/1052
Patching output file 943/1052
Patching output file 944/1052
Patching output file 945/1052
Patching output file 946/1052
Patching output file 947/1052
Patching output file 948/1052
Patching output file 949/1052
Patching output file 950/1052
Patching output file 951/1052
Patching output file 952/1052
Patching output file 953/1052
Patching output file 954/1052
Patching output file 955/1052
Patching output file 956/1052
Patching output file 957/1052
Patching output file 958/1052
Patching output file 959/1052
Patching output file 960/1052
Patching output file 961/1052
Patching output file 962/1052
Patching output file 963/1052
Patching output file 964/1052
Patching output file 965/1052
Patching output file 966/1052
Patching output file 967/1052
Patching output file 968/1052
Patching output file 969/1052
Patching output file 970/1052
Patching output file 971/1052
Patching output file 972/1052
Patching output file 973/1052
Patching output file 974/1052
Patching output file 975/1052
Patching output file 976/1052
Patching output file 977/1052
Patching output file 978/1052
Patching output file 979/1052
Patching output file 980/1052
Patching output file 981/1052
Patching output file 982/1052
Patching output file 983/1052
Patching output file 984/1052
Patching output file 985/1052
Patching output file 986/1052
Patching output file 987/1052
Patching output file 988/1052
Patching output file 989/1052
Patching output file 990/1052
Patching output file 991/1052
Patching output file 992/1052
Patching output file 993/1052
Patching output file 994/1052
Patching output file 995/1052
Patching output file 996/1052
Patching output file 997/1052
Patching output file 998/1052
Patching output file 999/1052
Patching output file 1000/1052
Patching output file 1001/1052
Patching output file 1002/1052
Patching output file 1003/1052
Patching output file 1004/1052
Patching output file 1005/1052
Patching output file 1006/1052
Patching output file 1007/1052
Patching output file 1008/1052
Patching output file 1009/1052
Patching output file 1010/1052
Patching output file 1011/1052
Patching output file 1012/1052
Patching output file 1013/1052
Patching output file 1014/1052
Patching output file 1015/1052
Patching output file 1016/1052
Patching output file 1017/1052
Patching output file 1018/1052
Patching output file 1019/1052
Patching output file 1020/1052
Patching output file 1021/1052
Patching output file 1022/1052
Patching output file 1023/1052
Patching output file 1024/1052
Patching output file 1025/1052
Patching output file 1026/1052
Patching output file 1027/1052
Patching output file 1028/1052
Patching output file 1029/1052
Patching output file 1030/1052
Patching output file 1031/1052
Patching output file 1032/1052
Patching output file 1033/1052
Patching output file 1034/1052
Patching output file 1035/1052
Patching output file 1036/1052
Patching output file 1037/1052
Patching output file 1038/1052
Patching output file 1039/1052
Patching output file 1040/1052
Patching output file 1041/1052
Patching output file 1042/1052
Patching output file 1043/1052
Patching output file 1044/1052
Patching output file 1045/1052
Patching output file 1046/1052
Patching output file 1047/1052
Patching output file 1048/1052
Patching output file 1049/1052
Patching output file 1050/1052
Patching output file 1051/1052
Patching output file 1052/1052
type lookup cache used 6335/65536 hits=14996 misses=7844
symbol lookup cache used 4945/65536 hits=27893 misses=4945
finished...
[ 74%] Built target docs
make: *** [all] Error 2

HOMEBREW_VERSION: 4.3.17-44-g474f576
ORIGIN: https://github.com/Homebrew/brew
HEAD: 474f5763f01a608830701f0c08d56a6b46191135
Last commit: 71 minutes ago
Core tap JSON: 20 Aug 13:22 UTC
Core cask tap JSON: 20 Aug 13:22 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_NO_BOOTSNAP: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 16-core 64-bit arm_palma
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.9.1 => /opt/homebrew/opt/curl/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
CXXFLAGS: -I/opt/homebrew/opt/icu4c/include
LDFLAGS: -L/opt/homebrew/opt/icu4c/lib
MAKEFLAGS: -j16
CMAKE_PREFIX_PATH: /opt/homebrew/opt/curl:/opt/homebrew/opt/icu4c:/opt/homebrew
CMAKE_INCLUDE_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /opt/homebrew/opt/graphviz/lib/pkgconfig:/opt/homebrew/opt/brotli/lib/pkgconfig:/opt/homebrew/opt/libidn2/lib/pkgconfig:/opt/homebrew/opt/libnghttp2/lib/pkgconfig:/opt/homebrew/opt/openssl@3/lib/pkgconfig:/opt/homebrew/opt/libssh2/lib/pkgconfig:/opt/homebrew/opt/rtmpdump/lib/pkgconfig:/opt/homebrew/opt/lz4/lib/pkgconfig:/opt/homebrew/opt/xz/lib/pkgconfig:/opt/homebrew/opt/zstd/lib/pkgconfig:/opt/homebrew/opt/curl/lib/pkgconfig:/opt/homebrew/opt/miniupnpc/lib/pkgconfig:/opt/homebrew/opt/opus/lib/pkgconfig:/opt/homebrew/opt/icu4c/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/opt/homebrew/Library/Homebrew/os/mac/pkgconfig/14
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk
ACLOCAL_PATH: /opt/homebrew/opt/curl/share/aclocal:/opt/homebrew/share/aclocal
PATH: /opt/homebrew/Library/Homebrew/shims/mac/super:/opt/homebrew/opt/cmake/bin:/opt/homebrew/opt/doxygen/bin:/opt/homebrew/opt/graphviz/bin:/opt/homebrew/opt/node/bin:/opt/homebrew/opt/pkg-config/bin:/opt/homebrew/opt/brotli/bin:/opt/homebrew/opt/gettext/bin:/opt/homebrew/opt/libidn2/bin:/opt/homebrew/opt/openssl@3/bin:/opt/homebrew/opt/rtmpdump/bin:/opt/homebrew/opt/lz4/bin:/opt/homebrew/opt/xz/bin:/opt/homebrew/opt/zstd/bin:/opt/homebrew/opt/curl/bin:/opt/homebrew/opt/miniupnpc/bin:/opt/homebrew/opt/icu4c/bin:/usr/bin:/bin:/usr/sbin:/sbin
LIBRARY_PATH: /opt/homebrew/opt/icu4c/lib

```

Last 15 lines from /Users/wwt13/Library/Logs/Homebrew/sunshine-beta/02.make:
Patching output file 1043/1052
Patching output file 1044/1052
Patching output file 1045/1052
Patching output file 1046/1052
Patching output file 1047/1052
Patching output file 1048/1052
Patching output file 1049/1052
Patching output file 1050/1052
Patching output file 1051/1052
Patching output file 1052/1052
type lookup cache used 6338/65536 hits=14998 misses=7847
symbol lookup cache used 4948/65536 hits=27920 misses=4948
finished...
[ 54%] Built target docs
make: *** [all] Error 2
Getting the same error as @RitwickVerma on an M1 Mac.

--- a/src/platform/common.h
+++ b/src/platform/common.h
@@ -11,6 +11,10 @@
 #include <string>
 
 #include <boost/core/noncopyable.hpp>
+#ifndef _WIN32
+#include <boost/process.hpp>
+#include <boost/asio.hpp>
+#endif
 
 #include "src/config.h"
 #include "src/logging.h"
@@ -31,6 +35,7 @@ struct AVHWFramesContext;
 struct AVCodecContext;
 struct AVDictionary;
 
+#ifdef _WIN32
 // Forward declarations of boost classes to avoid having to include boost headers
 // here, which results in issues with Windows.h and WinSock2.h include order.
 namespace boost {
@@ -50,6 +55,7 @@ namespace boost {
     typedef basic_environment<char> environment;
   }  // namespace process
 }  // namespace boost
+#endif
 namespace video {
   struct config_t;
 }  // namespace video

--- a/src/platform/common.h
+++ b/src/platform/common.h
@@ -12,8 +12,8 @@
 
 #include <boost/core/noncopyable.hpp>
 #ifndef _WIN32
-#include <boost/process.hpp>
-#include <boost/asio.hpp>
+  #include <boost/asio.hpp>
+  #include <boost/process.hpp>
 #endif
 
 #include "src/config.h"

